docs/static-header-footer/site-assets/x-bootstrap-4.css

```diff
@@ -263,6 +263,12 @@ a.btn, .nav a {
 	margin-top: 20px;
 	width: auto;
 }
+@media screen and (min-width: 480px) {
+	.colcount-xs-2 {
+		-webkit-column-count: 2;
```

Comment:
I do think our post-css will fix this browser CSS

Reply:
Do you mean that autoprefixer might do it as a part of the postcss task? I just tried removing that line and ran grunt to no avail (the line wasn't re-inserted).

PostCSS only deals with the dist folder's compiled CSS files. This experimental Bootstrap 4 file isn't situated there. It's a "loose" CSS file in the repo's source files.